tox.ini

```diff
@@ -50,6 +50,7 @@ extras =
 commands_pre =
     oldestdeps: minimum_dependencies mpl_animators --filename requirements-min.txt
     oldestdeps: pip install -r requirements-min.txt
+    oldestdeps: pip install "astropy<6.0.0"
```

Comment:
minimum_dependencies only seems to care about core packages, not optional ones.

Reply:
You can give it --extras